pyiceberg/io/fsspec.py

```diff
@@ -175,14 +184,50 @@ def _gs(properties: Properties) -> AbstractFileSystem:
 def _adlfs(properties: Properties) -> AbstractFileSystem:
     from adlfs import AzureBlobFileSystem
 
+    for property_name in properties:
+        if property_name.startswith(ADLFS_PREFIX):
+            deprecation_message(
+                deprecated_in="0.7.0",
```

Comment:
nit: 0.7.1 (or whenever this PR gets in)

Reply:
Fixed